src/identity/reducer.ts

```diff
@@ -257,6 +264,22 @@ export const reducer = (
         e164NumberToSalt: state.e164NumberToSalt,
         secureSendPhoneNumberMapping: state.secureSendPhoneNumberMapping,
       }
+    case Actions.FETCH_ADDRESS_VERIFICATION:
+      return {
+        ...state,
+        addressToVerificationStatus: {
+          ...state.addressToVerificationStatus,
+          [action.address]: !state.addressToVerificationStatus[action.address] ? undefined : true,
```

Comment:
nit: 
```suggestion
          [action.address]: state.addressToVerificationStatus[action.address] || undefined
```
Also maybe leave a comment on why its set to undefined if `false`?